app/components/number-widget/component.js

```diff
@@ -7,6 +7,7 @@ export default Ember.Component.extend({
     settings : {
         fontSize: 3,
         fontColor: '#F44336',
+        noLink: false,
```

Comment:
You would only need to change it in the `number-widget` components right? But I see what you mean where having the link is essentially default unless you specify "noLink"